src/i18n/en/nav.ts

```diff
@@ -111,6 +111,11 @@ export default [
 		slug: 'reference/directives-reference',
 		key: 'reference/directives-reference',
 	},
+	{
+		text: 'Built-in components',
```

Comment:
```suggestion
		text: 'Built-in Components',
```
There once was a letter quite small,
That slipped through a crack in the wall.
It got lost in the code,
Walked many a road,
Until Vox made it quite a bit tall.

Reply:
A wild Voxel appears,
Feels like hasn't been spotted in years!
Cut to the chase,
With a fix to the case,
To a resounding chorus of cheers! 🎉